Feature/pg/fix 791 doc (#5)
SonarSource/sonarqube-docker-docs: sonarqube/content.md

Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.

### Use bind-mounted folders or volumes

> -	SonarQube 7.9.X LTS : Use volumes

Author: ```suggestion
```

Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.

### Use bind-mounted folders or volumes

> -	SonarQube 8.X : Use bind-mounted folders

Author: ```suggestion
```

-	SonarQube 7.9.X LTS : Use volumes

The images contain the SonarQube installation at `/opt/sonarqube`. You need to bind them to your host to override selected files or directories :

-	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
-	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
-	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
-	`/opt/sonarqube/extensions`: plugins, such as language analyzers

### SonarQube 7.9.X LTS First installation

Follow these steps for your first installation:

> 1.	Create volumes `sonarqube_conf`, `sonarqube_data`, `sonarqube_logs` and `sonarqube_extensions` and start the image with the following command. This will populate all the volumes (copying default plugins, create the elasticsearch data folder, create the sonar.properties configuration file). Watch the logs, and once the container is properly started you can force-exit (ctrl+c) and proceed to the next step.

Reviewer: ```suggestion
1.	Create volumes `sonarqube_conf`, `sonarqube_data`, `sonarqube_logs`, and `sonarqube_extensions` and start the image with the following command. This will populate all the volumes (copying default plugins, create the Elasticsearch data folder, create the sonar.properties configuration file). Watch the logs, and, once the container is properly started, you can force-exit (ctrl+c) and proceed to the next step.
```

	```console
	$ docker run --rm \
	    -p 9000:9000 \
	    -v sonarqube_conf:/opt/sonarqube/conf \
	    -v sonarqube_extensions:/opt/sonarqube/extensions \
	    -v sonarqube_logs:/opt/sonarqube/logs \
	    -v sonarqube_data:/opt/sonarqube/data \
	    %%IMAGE%%
	```

2.	Configure sonar.properties to configure the database JDBC URL. Templates are available for every supported database. Just uncomment and configure the template you need. Please note that due to [SONAR-12501](https://jira.sonarsource.com/browse/SONAR-12501), providing `sonar.jdbc.username` and `sonar.jdbc.password` in `sonar.properties` is not working, and you will need to explicitly define theses values in the docker run command with the `-e` flag.

	```plain
	#Example for PostgreSQL
	sonar.jdbc.url=jdbc:postgresql://localhost/sonarqube
	```

3.	Drivers for the supported databases (except Oracle) are already provided. Do not replace the provided drivers; they are the only ones supported. For Oracle, copy the JDBC driver into `$SONARQUBE_HOME/extensions/jdbc-driver/oracle`.

> 4.	Run the image with your jdbc username and password :

Reviewer: ```suggestion
4.	Run the image with your JDBC username and password :
```

	    %%IMAGE%%
	```

## Upgrade SonarQube 7.9.X LTS to another 7.9.X LTS version

> Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). No specific docker operation are needed, just use the new tag.

Reviewer: ```suggestion
Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). No specific Docker operations are needed, just use the new tag.
```

## Upgrade from SonarQube 7.9.X LTS to SonarQube 8.X and later

> Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). On top of that, you need to switch from volumes to host-binded folder for easier next upgrade procedure. Follow the next chapiter `Upgrade from SonarQube 8.X and later` instructions, but instead of upgrading from a backup-folder to a new folder, you will upgrade from your backup-volume to a new folder. Starting with the 8.0 version, You can move your `sonar.jdbc.username` and `sonar.jdbc.password` value to `sonar.properties` instead of passing them with the `-e` flag.

Reviewer: ```suggestion
Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). In addition, you need to switch from volumes to host-binded folder to make the next upgrade step easier. Follow the instructions in the next section `Upgrade from SonarQube 8.X and later` , but instead of upgrading from a backup-folder to a new folder, you will upgrade from your backup-volume to a new folder. Starting with the version 8.0, you can move your `sonar.jdbc.username` and `sonar.jdbc.password` value to `sonar.properties` instead of passing them with the `-e` flag.
```

Reviewer: Is host-binded the same thing as bind-mounted folders? If so, I would use bind-mounted since that's the terminology used earlier.

Author: yes it is !
